lib/nile/src/Nile.ts

```diff
@@ -13,18 +14,62 @@ import {
   ConfigurationParameters,
 } from './generated/openapi/src/runtime';
 
+export class EventsApi {
+  static onCounter: number = 0;
+  entities: EntitiesApi;
+  timers: { [key: string]: NodeJS.Timer };
```

Comment:
I think this is probably a number for the key, and `NodeJS.Timer` feels weird for something that could be in the browser. I am not sure if there's a more agnostic `TimerId` or something.

Reply:
TIL: `ReturnType<typeof setTimeout>`